### PR TITLE
Update managing_permissions.md

### PR DESCRIPTION
**Pull request by maedelange**

### CODE REVIEW

File: source/managing_permissions.md [hunk 1]
```diff
@@ -16,22 +16,22 @@ to collaborate on your projects.
...
+form will require authentication for an individual to be able to access the form and send submissions to the project.
 
-In some cases, you may wish to limit data entry to require that individuals be
+In some cases, you may wish to enable data entry from anyone who knows the Enketo web form URL. In this case, any individual with access to the form URL can send submissions to the project, without them being required to be
 logged into their KoboToolbox user accounts.
 
-- Go to **ACCOUNT SETTINGS** inside your KoboToolbox account
-- Check "Require authentication to see forms and submit data"
+- Go to **Collect data** inside your KoboToolbox account
+- Select "Allow web submissions to this form without a username and password"
```
Comment (noliveleger) [inline source/managing_permissions.md:27]: `web` could removed (according to loom video).

File: source/managing_permissions.md [hunk 2]
```diff
@@ -16,22 +16,22 @@ to collaborate on your projects.
...
 
 ![Requiring passwords](images/managing_permissions/requiring_passwords.gif)
 
+You can also access this by going to the **Sharing** tab in the **SETTINGS** page in your data collection project and toggling between the default setting to “Allow web submissions to this form without a username and password”.
```
Comment (noliveleger) [inline source/managing_permissions.md:31]: same here . remove `web` in web submissions.

File: source/managing_permissions.md [hunk 3]
```diff
@@ -16,22 +16,22 @@ to collaborate on your projects.
...
-In some cases, you may wish to limit data entry to require that individuals be
+In some cases, you may wish to enable data entry from anyone who knows the Enketo web form URL. In this case, any individual with access to the form URL can send submissions to the project, without them being required to be
 logged into their KoboToolbox user accounts.
 
-- Go to **ACCOUNT SETTINGS** inside your KoboToolbox account
-- Check "Require authentication to see forms and submit data"
+- Go to **Collect data** inside your KoboToolbox account
+- Use the toggle button to enable "Allow submissions to this form without a username and password"
 
 ![Requiring passwords](images/managing_permissions/requiring_passwords.gif)
```
Comment (exprin) [inline source/managing_permissions.md:29]: Permissions

File: source/managing_permissions.md [hunk 4]
```diff
@@ -16,22 +16,22 @@ to collaborate on your projects.
...
+ managing your projects, and submitting data. When a new project has been deployed, the Enketo web
+form will require authentication for an individual to be able to access the form and send submissions to the project.
 
-In some cases, you may wish to limit data entry to require that individuals be
+In some cases, you may wish to enable data entry from anyone who knows the Enketo web form URL. In this case, any individual with access to the form URL can send submissions to the project, without them being required to be
 logged into their KoboToolbox user accounts.
 
-- Go to **ACCOUNT SETTINGS** inside your KoboToolbox account
-- Check "Require authentication to see forms and submit data"
+- Go to **Collect data** inside your KoboToolbox account
```
Comment (exprin) [inline source/managing_permissions.md:26]: - Open the project in KoboToolbox, and go to the FORM section

File: source/managing_permissions.md [hunk 5]
```diff
@@ -16,22 +16,22 @@ to collaborate on your projects.
...
+- Use the toggle button to enable "Allow submissions to this form without a username and password"
```
Comment (exprin) [inline source/managing_permissions.md:27]: - Below the Collect Data section, use the toggle button to enable "Allow submissions to this form without a username and password"

File: source/managing_permissions.md [hunk 6]
```diff
@@ -16,22 +16,22 @@ to collaborate on your projects.
...
 
 ![Requiring passwords](images/managing_permissions/requiring_passwords.gif)
 
+You can also make changes to this privacy feature by going to the **Sharing** tab in the **SETTINGS** page in your data collection project and toggling the default setting to enable “Allow submissions to this form without a username and password”, or when accessing your **Sharing Permissions**.
+
```
Comment (exprin) [inline source/managing_permissions.md:32]: Sharing settings

**Review by exprin — CHANGES_REQUESTED** [review]

**Pull request merged**